fvtest/tril/tril/CMakeLists.txt

```diff
@@ -23,44 +23,88 @@ set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../cmake/modules"
 	${CMAKE_MODULE_PATH})
 
 include(OmrUtility)
-include(FindYACC)
-include(FindLEX)
-
-find_package(YACC REQUIRED)
-find_package(LEX REQUIRED)
 
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-list(APPEND TRIL_FLEX_FLAGS)
+find_package(YACC)
+find_package(LEX)
+
+if (YACC AND LEX)
```

Comment:
YACC and LEX, aren't defined anywhere. The variables you should be testing are `YACC_EXECUTABLE` and `LEX_EXECUTABLE`. It would be better if the find modules defined `YACC_FOUND`/`LEX_FOUND`. I'll open an issue for that as well.

Reply:
For reference #4070 was opened and fixed in #4071. Thanks @rwy0717!